test/view_test.py

```diff
@@ -236,6 +236,14 @@ def test_write_nonstr(self):
         self.view[0][4] = 'nya'
         self.view[1][0] = Dummy()
         self.view[1][4] = None
+
+        self.assertEqual(self.view[0][0], 28)
+        self.assertEqual(self.view[0][1], 28.3)
```

Comment:
While this tests that the data type hasn't been modified upon modifying the view, it would be nice to also check if the actual value in the sheet matches. Could you add a test to that effect?